### DISCUSSION TITLE
[Feature] - Time limit for multiplayer mode

### DISCUSSION THREAD

**Discussion by jevonmao** [pointfreeco/isowords#89] [Ideas]
As described in the title, I think it will be a good feature if we add a time limit for each turn in multiplayer mode. For example, there can be an adjustable time limit of say 1 minute before the person's turn is forfeited. 

I will add more design details because I would actually like to work on this, once I get a better understanding of the codebase (just discovered this amazing project today).

**Comment by stephencelis** [comment]
😄 It's definitely a fun idea we've discussed in the past! Just haven't gotten around to trying to implement it.
